djpaddle/models.py

```diff
@@ -191,8 +194,7 @@ def from_subscription_created(cls, payload):
     @classmethod
     def update_by_payload(cls, payload):
         data = cls._sanitize_webhook_payload(payload)
-        pk = data.pop("id")
-        return cls.objects.update_or_create(pk, defaults=data)
+        return cls.objects.update_or_create(id=data["id"], defaults=data)
```

Comment:
update_or_create

Reply:
data.pop is used here to make sure the ID is not part of the defaults